wsi/physicaldisplay.cpp

```diff
@@ -37,8 +37,10 @@ PhysicalDisplay::PhysicalDisplay(uint32_t gpu_fd, uint32_t pipe_id)
     : pipe_(pipe_id),
       width_(0),
       height_(0),
+      custom_resolution_(false),
       gpu_fd_(gpu_fd),
       power_mode_(kOn) {
+  rect_.left = rect_.top = rect_.right = rect_.bottom = 0;
```

Comment:
It's initialized to zero, no need to do again

Reply:
Should I remove? I was not sure because even the width_ and height_ were getting initialized to 0.

Reply:
ya HWCRect initializes everything to 0 in constructor

Reply:
Removed initialization to 0